opendevin/sandbox/sandbox.py

```diff
@@ -70,12 +76,13 @@ def read_logs(self) -> str:
         return logs
 
     def execute(self, cmd: str) -> Tuple[int, str]:
-        exit_code, logs = self.container.exec_run(['/bin/bash', '-c', cmd], workdir="/workspace")
+        # TODO: each execute is not stateful! We need to keep track of the current working directory
+        exit_code, logs = self.container.exec_run(['su', 'devin', '-c', cmd], workdir="/workspace")
         return exit_code, logs.decode('utf-8')
 
     def execute_in_background(self, cmd: str) -> None:
         self.log_time = time.time()
-        result = self.container.exec_run(['/bin/bash', '-c', cmd], socket=True, workdir="/workspace")
+        result = self.container.exec_run(['su', 'devin', '-c', cmd], socket=True, workdir="/workspace")
         self.log_generator = result.output # socket.SocketIO
```

Comment:
I do this to fix the permission issue - `/bin/bash` will run `cmd` as root, which make command like `chmod +x` failed

Reply:
Let's use #124 to fix it :)